syncToTestApp.ps1

```diff
@@ -0,0 +1,38 @@
+
+if( $args[0] -ne $null ) {
```

Comment:
I think we should be consistent on the spaces before and after parens. Personally, I'd prefer a space after `if` and not spaces inside the parens if that's OK in Powershell.

Reply:
sure. I don't think there's a convention for powershell, but it's syntactically correct, so I'll update it.